backend/backend/src/main/java/de/aaaaaaah/velcom/backend/access/KnownCommitWriteAccess.java

```diff
@@ -24,17 +24,17 @@ public KnownCommitWriteAccess(DatabaseStorage databaseStorage, TaskWriteAccess t
 	public void markCommitsAsKnownAndInsertIntoQueue(RepoId repoId, Collection<CommitHash> commits,
 		Collection<Task> tasks) {
 
-		databaseStorage.acquireTransaction(db -> {
+		databaseStorage.acquireWriteTransaction(db -> {
 			markCommitsAsKnown(repoId, commits, db);
 			taskAccess.insertTasks(tasks, db);
 		});
 	}
 
 	public void markCommitsAsKnown(RepoId repoId, Collection<CommitHash> commits) {
-		markCommitsAsKnown(repoId, commits, databaseStorage.acquireContext());
+		markCommitsAsKnown(repoId, commits, databaseStorage.acquireWriteAccess());
```

Comment:
Why doesn't this use try-with-resources?

Reply:
Fixed in https://github.com/IPDSnelting/velcom/commit/9c85694d224e55f310d9330460dca600ea140c58